src/components/implementation/no_interface/events_prototype/Makefile

```diff
@@ -6,10 +6,15 @@ DEPENDENCIES=
 ADDITIONAL_LIBS=-lcobj_format -lcos_defkernel_api -lcos_kernel_api -lsl -lheap -lsl_thd_static_backend -lsl_lock -lsl_mod_fprr
 OBJLIBS += $(POSIX_LIB)
 
-export XARGO_RUST_SRC=$(shell rustc --print sysroot)/lib/rustlib/src/rust/src/
-export RUSTFLAGS=-L ../../../../lib/musl-1.1.11/lib/ -L lib
 CRATE_DIR=events
 
+XARGO_RUST_SRC=$(shell rustc --print sysroot)/lib/rustlib/src/rust/src/
+RUSTFLAGS=-L $(shell pwd)/../../../lib/musl-1.1.11/lib/ -L $(shell pwd)/$(CRATE_DIR)/lib
+RUST_TARGET_PATH=$(shell pwd)/$(CRATE_DIR)
+export XARGO_RUST_SRC
+export RUSTFLAGS
+export RUST_TARGET_PATH
```

Comment:
Are you sure these modifications are right? Have you tested these changes on ZSH Ubuntu 16, bash Ubuntu 16, and bash Ubuntu 14? Sorry to be nitpicky, but I don't want Rust to be broken on any platform. 

Also, why split up the exports onto their own lines?

Reply:
I don't think that @RobertGiff has any context for Rust.  I honestly (wrongly) assumed that you had provided these changes for him as they seem oddly specific.

Reply:
I made these changes so that it would compile on 16.04 ZSH. @Others it does also work on 16.04 Bash. I unfortunately don't have a set up for Ubuntu 14 currently, if you pull and it doesn't work for some reason on Ubuntu 14, revert back to old form until a compatible fix is put in.